Formula/rom-tools.rb

```diff
@@ -23,13 +23,16 @@ class RomTools < Formula
   depends_on "sdl2"
   depends_on "utf8proc"
 
+  uses_from_macos "expat"
+  uses_from_macos "zlib"
+
   def install
     inreplace "scripts/src/osd/sdl.lua", "--static", ""
     system "make", "TOOLS=1",
                    "USE_LIBSDL=1",
                    "USE_SYSTEM_LIB_EXPAT=1",
                    "USE_SYSTEM_LIB_ZLIB=1",
-                   "USE_SYSTEM_LIB_ASIO=0",
```

Comment:
is this change desired?

Reply:
In `USE_SYSTEM_LIB_ASIO`, no.
But in `USE_SYSTEM_LIB_LUA`, set blank value is required.

Reply:
yeah, I was thinking that this build option should be set with some value.

Reply:
In most `USE_SYSTEM_LIB` flags, it can set `0` or blank value to disable it.
But `USE_SYSTEM_LIB_LUA`, it must set blank value to disable, set `0` is broken.
So in my commit, `USE_SYSTEM_LIB_ASIO`'s value is set like `USE_SYSTEM_LIB_LUA`.

Reply:
Make sense, thanks!